### ISSUE TITLE
ArchWSL blocks startup of Docker Desktop for Windows 4.1.0 

### ISSUE THREAD

**Issue by danielwagn3r** [yuk7/ArchWSL#247]
**IMPORTANT**
Please read [README](https://github.com/yuk7/ArchWSL/blob/master/README.md) and [Docs](https://git.io/arch-doc) before creating the issue.

**Please fill out the below information:**
**Describe the issue**
Docker Desktop for Windows 4.1.0 doesn't startup when ArchWSL on WSL2 is installed on the system.

**To Reproduce**
- Install WSL2
- Install ArchWSL
- Install Docker Desktop for Windows 4.1.0

**Expected behavior**
Coexistense of ArchWSL and Docker Desktop for Win wasn't a problem previously.

**Screenshots**

**Enviroment:**
 - Windows build number: Microsoft Windows [Version 10.0.19043.1237]
 - Security Software: [e.g. ESET Internet Security]
 - WSL version 1/2: WLS2
 - ArchWSL version 21.8.28.0
 - ArchWSL Installer type appx
 - Launcher version wsldl2, version 21082800  (amd64)

**Additional context**
See https://github.com/docker/for-win/issues/12229

**Comment by yuk7** [comment]
Are you configured to use the WSL2 engine in Docker Desktop?

**Comment by yuk7** [comment]
Did you configure it to use the WSL2 engine on your Docker desktop?

**Comment by danielwagn3r** [comment]
> Did you configure Docker desktop to use the WSL2 engine?

Seems Docker Inc. has found an issue with a change in their build.

**Comment by yuk7** [comment]
I see,
It seems to be a issue on the Docker side.
They seem to be working on a fix.

**Comment by danielwagn3r** [comment]
> I see,
> It seems to be a issue on the Docker side.
> They seem to be working on a fix.

Yep, think this can be closed. There's also evidence that it's nothing ArchWSL specific.
